torch_xla/csrc/runtime/xla_coordinator.h

```diff
@@ -0,0 +1,58 @@
+#ifndef PTXLA_RUNTIME_COORDINATOR_H_
+#define PTXLA_RUNTIME_COORDINATOR_H_
+
+#include <memory>
+
+#include "tsl/distributed_runtime/preemption/preemption_sync_manager.h"
+
+// Forward declaration
```

Comment:
Same here. Do you need the forward declaration?

Reply:
I only need one to avoid the log macro issue, this one can be removed.